File: televisionbingecalculator/src/main/java/com/vanillax/televisionbingecalculator/app/ServerAPI/TV/Cast.java
package com.vanillax.televisionbingecalculator.app.ServerAPI.TV;

import com.google.gson.annotations.SerializedName;

/**
 * Created by mitchross on 5/21/17.
 */

public class Cast
{
	@SerializedName( "name" )
	public String name;

	@SerializedName( "profile_path" )
	public String profilePath;
}

File: televisionbingecalculator/src/main/java/com/vanillax/televisionbingecalculator/app/ServerAPI/TV/Cast.kt
package com.vanillax.televisionbingecalculator.app.ServerAPI.TV

import com.google.gson.annotations.SerializedName

/**
 * Created by mitchross on 5/21/17.
 */

class Cast {
    @SerializedName("name")
    var name: String? = null

    @SerializedName("profile_path")
    var profilePath: String? = null
}
